test/unit/trading/calculate-trade-cost.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-env mocha */
+
+"use strict";
+
+var assert = require("chai").assert;
+var calculateTradeCost = require("../../../src/trading/calculate-trade-cost");
+
+describe("trading/calculate-trade-cost", function () {
+  var test = function (t) {
+    it(t.description, function () {
+      t.assertions(calculateTradeCost(t.params));
+    });
+  };
+  test({
+    description: "scalar bid",
+    params: {
+      displayPrice: "20",
```

Comment:
Recommend one or more test cases with a negative `displayPrice`.

Reply:
Good thinking!  Do you want to write a couple?  (You know you do)